src/m365/spo/commands/page/page-set.ts

```diff
@@ -176,7 +189,7 @@ class SpoPageSetCommand extends SpoCommand {
         pageId = page.Id;
 
         bannerImageUrl = page.BannerImageUrl;
-        canvasContent1 = page.CanvasContent1;
+        canvasContent1 = canvasContent1 || page.CanvasContent1;
```

Comment:
I think it's clearer if we reference args.options.content here instead of up above.